### ISSUE TITLE
Uncaught TypeError: Path must be a string. Received true

### ISSUE THREAD

**Issue by ggranger** [armin-pfaeffle/sass-autocompile#22]
[Enter steps to reproduce below:]

1. ...
2. ...

**Atom Version**: 1.0.16
**System**: Mac OS X 10.10.5
**Thrown From**: [sass-autocompile](https://github.com/armin-pfaeffle/sass-autocompile) package, v0.10.1

### Stack Trace

Uncaught TypeError: Path must be a string. Received true

```
At path.js:8

TypeError: Path must be a string. Received true
    at assertPath (path.js:8:11)
    at Object.posix.resolve (path.js:413:5)
    at /Users/guillaume/.atom/packages/sass-autocompile/lib/inline-parameters.coffee:15:43
    at ReadStream.<anonymous> (/Users/guillaume/.atom/packages/sass-autocompile/lib/inline-parameters.coffee:39:21)
    at emitOne (events.js:77:13)
    at ReadStream.emit (events.js:169:7)
    at readableAddChunk (_stream_readable.js:146:16)
    at ReadStream.Readable.push (_stream_readable.js:110:10)
    at onread (fs.js:1749:12)
    at FSReqWrap.wrapper [as oncomplete] (fs.js:576:17)
```

### Commands

```
     -0:00.7.0 core:save (atom-text-editor.editor.is-focused)
```

### Config

```json
{
  "core": {
    "disabledPackages": [
      "metrics",
      "wrap-guide",
      "welcome"
    ],
    "projectHome": "/Users/guillaume/Projects",
    "ignoredNames": [
      ".git",
      ".DS_Store",
      "*.pyc"
    ],
    "themes": [
      "one-dark-ui",
      "monokai"
    ]
  },
  "sass-autocompile": {
    "notifications": "Notifications"
  }
}
```

### Installed Packages

```coffee
# User
atom-color-highlight, v3.0.9
atom-jinja2, v0.5.0
color-picker, v2.0.12
emmet, v2.3.12
jsonlint, v1.1.2
monokai, v0.18.0
php-twig, v4.0.0
pigments, v0.15.0
pretty-json, v0.4.1
project-manager, v2.5.2
sass-autocompile, v0.10.1
white-cursor, v0.6.0

# Dev
No dev packages
```

**Comment by armin-pfaeffle** [comment]
Hey!

Do you have an first-line-comments? Can you please post your files, so I can reproduce that bug?

**Comment by ggranger** [comment]
sure, here is a simple version of the that used to work yesterday

// out : ../css/main.css

body {
    font-family: sans-serif;
    padding: 0;
}

**Comment by ggranger** [comment]
Just found out i got a different error by removing the space between "out" and ":". Which led me to the deprecated "out" command. And now I just got it working again.

**Comment by armin-pfaeffle** [comment]
Great to hear, that it works now.

But that's a bug, I do not want to have ;) I will published a patch right now (verion 0.10.2). It's fixed there.
